test/versioned-external/external-repos.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+/**
+ * name: folder name to checkout the repo into.
+ * repository: repo URL to clone from.
+ * branch: branch to checkout
+ * additionalFiles: String array of files/folders to checkout in addition to lib and tests/versioned.
+ */
+const repos = [
```

Comment:
can you log a follow up ticket to remove the `/tests/versioned/*.js` from all the `files` arrays in package.json?

Reply:
definitely. wanted to first make sure we were ok with the approach and maybe see it exercised a few times in earnest to make sure things work smoothly.